pytorch/training/buildspec.yml

```diff
@@ -28,71 +28,71 @@ context:
       target: deep_learning_container.py
 
 images:
```

Comment:
Could you leave only SM gpu image on in the buildspec, which concerns model parallel?

Reply:
Yes!